extensions/netty/runtime/src/main/java/io/quarkus/netty/runtime/NettyRecorder.java

```diff
@@ -25,7 +25,9 @@ public void run() {
                 DefaultChannelId.newInstance();
                 if (System.currentTimeMillis() - start > 1000) {
```

Comment:
You should *always* use `nanoTime` for durations.  While it isn't super likely to cause a problem here, things like NTP might cause false positives.  But anyway it's a good habit to be in!

Reply:
Thanks for the hint, but I didn't touched that part of the code in this pr :stuck_out_tongue:

Reply:
Fair, I didn't "request changes" 🙂